migration/20221008202022-transaction-table.js

```diff
@@ -0,0 +1,29 @@
+const { MigrationInterface, QueryRunner } = require("typeorm");
+
+module.exports = class transactionTable20221008202022 {
+
+    async up(queryRunner) {
+        await queryRunner.query(
+            `create table \`transaction\`(
+                \`id\` char(36) not null,
+                \`transaction_date\` datetime not null,
+                \`transaction_type\` integer not null,
+                \`amount\` decimal(8,2) null,
+                \`description\` varchar(1000) null,
+                \`payment_method_id\` char(36) null,
+                \`transaction_card_id\` char(36) null,
```

Comment:
Sorry, I forgot to mention that. Please make `payment_method_id` and `transaction_card_id` required.
@RamaMahjoub

Reply:
👍🏻